src/icons/Valora.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
```

Comment:
seems like something that should go in the branding repo, also could use a better name.

Reply:
Is there a way to exclude the `Valora` text from the image and use the translations for that text? Otherwise, this should probably go into the branding repo.